### DISCUSSION TITLE
PSADT will often freeze at "Initialization"

### DISCUSSION THREAD

**Discussion by tokenwill** [PSAppDeployToolkit/PSAppDeployToolkit#817] [Q&A]
I have been using PSADT for some time with little issues. Lately when deploying through sccm, the script will not run.

In the log file it will show "Deployment type is: Uninstall", then never move on to initiate the script.

This was originally only happening for uninstalls. Now we are running into the issue were more than half the attempts will also freeze at "Deployment type is: Installation" then time out without ever moving to the next line.

I can run the script as an admin user and it will run through the script without any issues but running in system context, it freezes at those line.

Any information would be super helpful. Thanks

**Comment by mjr4077au** [comment]
Please review this in PSADT v4 when it's released this year, the init code is heavily refactored.
